malasakit-django/pcari/migrations/0053_comment_nullable.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-06-05 19:35
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pcari', '0052_auto_20170609_1902'),
+    ]
+
+    operations = [
+        migrations.AlterField('Comment', 'message', models.TextField(blank=True, null=True))
```

Comment:
This is redundant--for strings, [a blank string is also a null string](https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.Field.null). (This is probably due to the fact that in many contexts, strings are terminated with a null character.) Therefore, this migration isn't necessary.